general/networking/traefik.md

```diff
@@ -57,6 +66,7 @@ services:
       traefik.frontend.headers.customResponseHeaders: X-Robots-Tag:noindex,nofollow,nosnippet,noarchive,notranslate,noimageindex
       traefik.frontend.headers.frameDeny: "true"
       traefik.frontend.headers.customFrameOptionsValue: 'allow-from https://example.com'
+#      traefik.frontend.auth.basic.users: xxx:xxx
```

Comment:
Note that this would break some clients that doesn't know how to deal with `http basic auth` like AndroidTV. Should probably be mentioned somewhere.

Reply:
Not in this instance. This is basic auth for traefik's dashboard, not jellyfin. I haven't added auth rules to jellyfin yet specifically for that reason.